agent/util-scripts/pbench-sysinfo-dump

```diff
@@ -123,6 +123,58 @@ function collect_ara_data {
 	fi
 }
 
+function collect_stockpile_data {
+	# parse config
+	stockpile_path=$(getconf.py stockpile_path stockpile)
+	stockpile_user=$(getconf.py stockpile_user stockpile)
+	stockpile_local_remote_user=$(getconf.py local_remote_user stockpile)
+	stockpile_host_remote_user=$(getconf.py host_remote_user stockpile)
+	stockpile_output_path=$(getconf.py stockpile_output_path stockpile)
+	stockpile_dependencies=$(getconf.py -l stockpile_dependencies packages)
+
+	# install stockpile and its dependencies
+	for package in $stockpile_dependencies; do
+       		which $package &>/dev/null
+        	if [[ $? != 0 ]]; then
+                	debug_log "[$script_name] $package is not installed on the system, installing"
+                	echo "$package is not installed on the system, installing"
+                	yum install $package -y &>/dev/null
+        	fi
+	done
+	if [[ -z "$stockpile_path" ]]; then
+        	stockpile_path=/var/lib/pbench-agent
+	fi
+	if [[ -d "$stockpile_path/stockpile" ]]; then
+        	rm -rf $stockpile_path/stockpile
+	fi
+	git clone https://github.com/redhat-performance/stockpile.git $stockpile_path/stockpile &>/dev/null
```

Comment:
Can we use `curl` instead to fetch a release tarball from github?

Reply:
We don't have a release for stockpile as of now, it would be nice to have a release and maybe an rpm for a stable release like what we have in pbench.

Reply:
you can add TODO here to handle this, once it's ready.
i'll also open issue for that.

Reply:
Seems best to work with `git submodules` [1](https://git-scm.com/docs/git-submodule).  Instead of cloning here, when we package the `pbench-agent` we should pull a specific version of `stockpile` into the source tree so that it ends up installed in `/opt/pbench-agent/...` somewhere.